test/function/samples/tree-shake-map-constructor/main.js

```diff
@@ -0,0 +1,12 @@
+assert.ok(
```

Comment:
This test was already working in current Rollup: 
https://rollupjs.org/repl/?version=3.20.7&shareable=JTdCJTIyZXhhbXBsZSUyMiUzQW51bGwlMkMlMjJtb2R1bGVzJTIyJTNBJTVCJTdCJTIyY29kZSUyMiUzQSUyMmFzc2VydC5vayglNUNuJTVDdG5ldyUyME1hcCglN0IlNUNuJTVDdCU1Q3QlNUJTeW1ib2wuaXRlcmF0b3IlNUQoKSUyMCU3QiU1Q24lNUN0JTVDdCU1Q3RyZXR1cm4lMjAlN0IlNUNuJTVDdCU1Q3QlNUN0JTVDdG5leHQoKSUyMCU3QiU1Q24lNUN0JTVDdCU1Q3QlNUN0JTVDdGNvbnNvbGUubG9nKCdzaWRlJTIwZWZmZWN0JyklM0IlNUNuJTVDdCU1Q3QlNUN0JTVDdCU1Q3RyZXR1cm4lMjAlN0IlMjBkb25lJTNBJTIwdHJ1ZSUyMCU3RCUzQiU1Q24lNUN0JTVDdCU1Q3QlNUN0JTdEJTVDbiU1Q3QlNUN0JTVDdCU3RCUzQiU1Q24lNUN0JTVDdCU3RCU1Q24lNUN0JTdEKSU1Q24pJTNCJTIyJTJDJTIyaXNFbnRyeSUyMiUzQXRydWUlMkMlMjJuYW1lJTIyJTNBJTIybWFpbi5qcyUyMiU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJvdXRwdXQlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJlcyUyMiU3RCU3RCU3RA==

Also, you should not `console.log` in a `function` test as it produces unwanted output on the console when the tests are executed.
This would be a slightly modified version that your change should fix:
https://rollupjs.org/repl/?version=3.20.7&shareable=JTdCJTIyZXhhbXBsZSUyMiUzQW51bGwlMkMlMjJtb2R1bGVzJTIyJTNBJTVCJTdCJTIyY29kZSUyMiUzQSUyMmxldCUyMGVmZmVjdCUyMCUzRCUyMGZhbHNlJTNCJTVDbiU1Q25uZXclMjBNYXAoJTdCJTVDbiU1Q3QlNUJTeW1ib2wuaXRlcmF0b3IlNUQoKSUyMCU3QiU1Q24lNUN0JTVDdHJldHVybiUyMCU3QiU1Q24lNUN0JTVDdCU1Q3RuZXh0KCklMjAlN0IlNUNuJTVDdCU1Q3QlNUN0JTVDdGVmZmVjdCUyMCUzRCUyMHRydWUlM0IlNUNuJTVDdCU1Q3QlNUN0JTVDdHJldHVybiUyMCU3QiUyMGRvbmUlM0ElMjB0cnVlJTIwJTdEJTNCJTVDbiU1Q3QlNUN0JTVDdCU3RCU1Q24lNUN0JTVDdCU3RCUzQiU1Q24lNUN0JTdEJTVDbiU3RCklM0IlNUNuJTVDbmFzc2VydC5vayhlZmZlY3QpJTNCJTIyJTJDJTIyaXNFbnRyeSUyMiUzQXRydWUlMkMlMjJuYW1lJTIyJTNBJTIybWFpbi5qcyUyMiU3RCU1RCUyQyUyMm9wdGlvbnMlMjIlM0ElN0IlMjJvdXRwdXQlMjIlM0ElN0IlMjJmb3JtYXQlMjIlM0ElMjJlcyUyMiU3RCU3RCU3RA==

Reply:
Thanks for your review! I corrected this test.